bindings/python/fdb/impl.py

```diff
@@ -1580,6 +1580,7 @@ def run(self):
 
             _network_thread = NetworkThread()
             _network_thread.daemon = True
+            _network_thread.name = "fdb-network-thread"
```

Comment:
It appears that setting the Python thread name doesn't actually set the thread name with the OS such that we would see it in tools like `top`. It's not a huge deal and we can leave this, but maybe we can add a comment to indicate that this is a known limitation.

It does seem possible to set the OS thread name using various modules, but I'm not sure if it's worth it.

Reply:
Interesting, did you run locally to verify that, I'm interested in learning how you verified. I will look into modules fwiw and add a comment for now.

Reply:
Yeah, I did run a quick local experiment. I know that we use this thread name in Java despite it being too long for Linux, and I was curious how Python would handle it. Turns out it didn't set it in the OS at all, which seems to have been confirmed by some questions on StackOverflow.